tests/pathfinding/fixtures/network_service.py

```diff
@@ -204,12 +214,12 @@ def populate_token_network(
         for channel_id, (
             p1_index,
             p1_deposit,
-            _p1_transferred_amount,
+            p1_transferred_amount,
```

Comment:
We could change this to `capacity` to not have to do the calculations below.

Reply:
will do in the next PR